packages/experimental/Spinner/src/Spinner.types.tsx

```diff
@@ -1,4 +1,5 @@
-import type { TextProps, ViewProps } from 'react-native';
+import type { Animated, TextProps, ViewProps } from 'react-native';
+import type { SvgProps } from 'react-native-svg';
 
```

Comment:
This file be named as `Spinner.types.ts`, and same for the tokens files ( ts instead of tsx)

Reply:
done.